org.scala-ide.sdt.core/src/org/scalaide/core/internal/project/ScalaProject.scala

```diff
@@ -507,13 +507,13 @@ class ScalaProject private (val underlying: IProject) extends ClasspathManagemen
         try {
           super.save()
         } catch {
-          case e:IOException =>
+          case e: IOException =>
             logger.error(s"An Exception occured saving the project-specific preferences for ${underlying.getName()} ! Your settings will not be persisted. Please report !")
-            throw(e)
-          }
+            throw (e)
```

Comment:
I know I should be a scout but cannot fix all the world :). Anyway I fix it as well.